src/main/java/org/folio/linked/data/configuration/KafkaListenerConfiguration.java

```diff
@@ -53,4 +54,11 @@ public ConsumerFactory<String, DataImportEvent> dataImportEventConsumerFactory()
     return new DefaultKafkaConsumerFactory<>(config, new StringDeserializer(), deserializer);
   }
 
+  @Bean
+  public RetryTemplate retryTemplate(FolioKafkaProperties folioKafkaProperties) {
```

Comment:
There is the same bean defined in org.folio.spring.tools.config.RetryTemplateConfiguration,
we just need to activate it with 'folio.retry.enabled = true' property, right?

Reply:
That is correct. Good find. Updated code.